features/desktop/persistance.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Unity Persistance
```

Comment:
Could you change "persistance" to "persistence" everywhere?  Including filenames?

Reply:
done 👍